tests/acceptance/features/coreApiSharePublicLink1/changingPublicLinkShare.feature

```diff
@@ -171,36 +197,18 @@ Feature: changing a public link share
       | 1               | 200              |
       | 2               | 403              |
 
-
-  Scenario Outline: normal user removes password of a public link (invite only public link)
```

Comment:
is it duplicate scenario?

Reply:
We cannot add passwords for `internal` public links. Do we need such a test anymore?

Reply:
there should be a scenario in sharingNG suite